cli/commands/run_cmd.py

```diff
@@ -128,73 +148,71 @@ def run(files: tuple[str, ...], skip_files: tuple[str, ...], system_env: bool):
                 )
                 continue
             if str(filepath.resolve()) not in config_files_to_run:
                 config_files_to_run.append(str(filepath.resolve()))
 
     else:
-        click.echo("Processing provided configuration files and directories:")
+        log.info("Processing provided configuration files and directories:")
         processed_files = set()
         for path_str in files:
             path = Path(path_str)
             if path.is_dir():
-                click.echo(f"  Discovering YAML files in directory: {path}")
+                log.info("  Discovering YAML files in directory: %s", path)
                 for yaml_ext in ("*.yaml", "*.yml"):
                     for filepath in path.rglob(yaml_ext):
                         if filepath.name.startswith("_") or filepath.name.startswith(
                             "shared_config"
                         ):
-                            click.echo(
-                                f"  Skipping discovery: {filepath} (underscore prefix or shared_config)"
+                            log.info(
+                                "  Skipping discovery: %s (underscore prefix or shared_config)", filepath
                             )
                             continue
                         processed_files.add(str(filepath.resolve()))
             elif path.is_file():
                 if path.suffix in [".yaml", ".yml"]:
                     processed_files.add(str(path.resolve()))
                 else:
-                    click.echo(
-                        click.style(f"  Ignoring non-YAML file: {path}", fg="yellow")
+                    log.warning(
+                        "  Ignoring non-YAML file: %s", path
                     )
         config_files_to_run = sorted(list(processed_files))
 
     if skip_files:
-        click.echo(f"Applying --skip for: {skip_files}")
+        log.info("Applying --skip for: %s", skip_files)
         final_list = []
         skipped_basenames = [os.path.basename(s) for s in skip_files]
         for cf in config_files_to_run:
             if os.path.basename(cf) in skipped_basenames:
-                click.echo(
-                    f"  Skipping execution: {cf} (due to --skip)"
+                log.info(
+                    "  Skipping execution: %s (due to --skip)", cf
                 )
                 continue
             final_list.append(cf)
         config_files_to_run = final_list
 
     if not config_files_to_run:
-        click.echo(
-            click.style(
-                "No configuration files to run after filtering. Exiting.", fg="yellow"
-            )
+        log.warning(
+            "No configuration files to run after filtering. Exiting."
         )
         return 0
 
-    click.echo(click.style("Final list of configuration files to run:", bold=True))
+    log.warning("Final list of configuration files to run:")
```

Comment:
this should likely be INFO

Reply:
actually, maybe we should combine the log from lines 203 and those from line 205 in one cohesive statement.
We can use `\n` between each config file to maintain readability.